corehq/apps/linked_domain/static/linked_domain/js/domain_links.js

```diff
@@ -247,36 +245,49 @@ hqDefine("linked_domain/js/domain_links", [
         self.modelsToPush = ko.observableArray();
         self.buildAppsOnPush = ko.observable(false);
         self.pushInProgress = ko.observable(false);
+        self.shouldShowSelectedERMDomain = ko.observable(false);
+        self.shouldShowSelectedMRMDomain = ko.observable(false);
         self.enablePushButton = ko.computed(function () {
             return self.domainsToPush().length && self.modelsToPush().length && !self.pushInProgress();
         });
-        self.shouldShowSelectedERMDomain = ko.observable(false);
-        self.shouldShowSelectedMRMDomain = ko.observable(false);
+        self.containsLiteAndFullLinks = ko.computed(function () {
+            if (!self.parent.hasFullAccess) {
+                // should not contain both
+                return false;
+            }
+            const accessSet = new Set();
+            for (const link of Object.values(self.parent.domainLinksByNames())) {
```

Comment:
Can't you use `self.parent.domainLinks` instead of calling `values` on `domainLinksByNames`?

Reply:
Come to think of it, could this whole loop be shortened to this?
```
    return _.uniq(_.pluck(self.parent.domainLinks(), 'hasFullAccess')).length == 2;
```

Reply:
Ah that is so much nicer. I need to get more familiar with underscore.js 🙏🏻

Reply:
See fbc681ac645d9d1d5a2048db5f7d17aceaf32802